.travis.yml

```diff
@@ -2,6 +2,20 @@ language: go
 go:
 - 1.x
 sudo: false
+before_script:
+  - sudo wget https://github.com/haproxytech/dataplaneapi/releases/download/v1.2.5/dataplaneapi -O /usr/local/bin/dataplaneapi
+  - sudo chmod +x /usr/local/bin/dataplaneapi
+install:
+  - wget http://www.haproxy.org/download/2.0/src/haproxy-2.0.14.tar.gz
+  - tar -zxvf haproxy-2.0.14.tar.gz
+  - pushd haproxy-2.0.14 && make TARGET=linux-glibc USE_OPENSSL=1 USE_ZLIB=1 USE_PCRE=1 && sudo make install && popd
```

Comment:
This is not working as haproxy binary in not in the $PATH

Reply:
But the test tries to execute dataplane, and dataplane is already started?

`Failed executing dataplaneapi: exec: already started`

Haproxy binary is installed with `sudo make install` into `/usr/local/sbin`
```
'haproxy' -> '/usr/local/sbin/haproxy'
'doc/haproxy.1' -> '/usr/local/share/man/man1/haproxy.1'
```
The path contains `/usr/local/sbin` 
```
$ export PATH="/home/travis/gopath/bin:/home/travis/.gimme/versions/go1.14.2.linux.amd64/bin:/home/travis/bin:/home/travis/bin:/home/travis/.local/bin:/usr/local/lib/jvm/openjdk11/bin:/opt/pyenv/shims:/home/travis/.phpenv/shims:/home/travis/perl5/perlbrew/bin:/home/travis/.nvm/versions/node/v8.12.0/bin:/home/travis/.rvm/gems/ruby-2.5.3/bin:/home/travis/.rvm/gems/ruby-2.5.3@global/bin:/home/travis/.rvm/rubies/ruby-2.5.3/bin:/home/travis/gopath/bin:/home/travis/.gimme/versions/go1.11.1.linux.amd64/bin:/usr/local/maven-3.6.0/bin:/usr/local/cmake-3.12.4/bin:/usr/local/clang-7.0.0/bin:/usr/local/sbin:
```

Reply:
`exec: already started` there is something else: https://github.com/haproxytech/haproxy-consul-connect/pull/37

I did not pay attention, there is no need to call run on command since already run. It works sometimes because this is racy I suspect.

Rebase with https://github.com/haproxytech/haproxy-consul-connect/pull/37 and this error should disappear

Reply:
As demonstrated by the latest commit:

```
$ haproxy -vv
1465HA-Proxy version 2.0.14 2020/04/02 - https://haproxy.org/
1466Build options :
1467  TARGET  = linux-glibc
1468  CPU     = generic
1469  CC      = gcc
1470  CFLAGS  = -O2 -g -fno-strict-aliasing -Wdeclaration-after-statement -fwrapv -Wno-unused-label -Wno-sign-compare -Wno-unused-parameter -Wno-old-style-declaration -Wno-ignored-qualifiers -Wno-clobbered -Wno-missing-field-initializers -Wtype-limits
1471  OPTIONS = USE_PCRE=1 USE_OPENSSL=1 USE_ZLIB=1
1472
1473Feature list : +EPOLL -KQUEUE -MY_EPOLL -MY_SPLICE +NETFILTER +PCRE -PCRE_JIT -PCRE2 -PCRE2_JIT +POLL -PRIVATE_CACHE +THREAD -PTHREAD_PSHARED -REGPARM -STATIC_PCRE -STATIC_PCRE2 +TPROXY +LINUX_TPROXY +LINUX_SPLICE +LIBCRYPT +CRYPT_H -VSYSCALL +GETADDRINFO +OPENSSL -LUA +FUTEX +ACCEPT4 -MY_ACCEPT4 +ZLIB -SLZ +CPU_AFFINITY +TFO +NS +DL +RT -DEVICEATLAS -51DEGREES -WURFL -SYSTEMD -OBSOLETE_LINKER +PRCTL +THREAD_DUMP -EVPORTS```

The issue has to be in the tests, either they:
- should not try to start dataplane if it was started already
- stop dataplane as part of each test cleanup

Reply:
I think #37 will really fix this